nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -380,7 +380,7 @@ in
   config = {
 
     nix.binaryCachePublicKeys = [ "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" ];
-    nix.binaryCaches = [ "https://cache.nixos.org/" ];
```

Comment:
That's not a good idea, since there might be someone relying on that slash (cachix is).

Reply:
By relying, are you referring to https://github.com/cachix/cachix/blob/275f732676c06aa23ae6c93049c5e54cb12ce30c/cachix/src/Cachix/Client/InstallationMode.hs#L158, i.e. there will be two copies of the same binary cache listed in `/etc/nix/nix.conf`?